src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Filters/ResourceUrlFilter.cs

```diff
@@ -35,10 +35,10 @@ public ValueTask<FluidValue> ProcessAsync(FluidValue input, FilterArguments argu
                 resourcePath = _httpContextAccessor.HttpContext.Request.PathBase.Add(resourcePath.Substring(1)).Value;
             }
 
-            // Don't prefix cdn if the path is absolute, or is in debug mode.
+            // Don't prefix cdn if the path includes a protocol, i.e. is an external url, or is in debug mode.
             if (!_options.DebugMode
                 && !String.IsNullOrEmpty(_options.CdnBaseUrl)
-                && !Uri.TryCreate(resourcePath, UriKind.Absolute, out var uri))
+                && !resourcePath.Contains("//", StringComparison.OrdinalIgnoreCase))
```

Comment:
`Contains` or `Startswith`

Reply:
And even if you mean `https://` I'd rather check `Starswith("http://") || Startswith("https://")`

Reply:
Was allowing for 
- `https://`
- `http://` or
- `//` i.e. protocol agnostic

if you prefer, can do `StartsWith` for all 3

Reply:
Please, I fear unknown consequences if a url had a `//` in the middle. Beginning with the most obvious, `https://`

Reply:
You can make a new local private method to make it simpler